code/models/projection_helper_functions.R

```diff
@@ -207,6 +207,11 @@ compute_predicted_cum_cases <- function(full_data, model, policy_variables_used,
       magrittr::not() %>% 
       all()
   })
+  # no_policy_counterfactual_data_storage %>% 
```

Comment:
Do we want this comment block added?

Reply:
@kendonB yeah, can this comment block be deleted?